wsmaster/che-core-api-machine/src/test/java/org/eclipse/che/api/machine/server/MachineManagerTest.java

```diff
@@ -91,8 +99,7 @@
     public void setUp() throws Exception {
         final SnapshotDao snapshotDao = mock(SnapshotDao.class);
         final EventService eventService = mock(EventService.class);
-        final String machineLogsDir = targetDir().resolve("logs-dir").toString();
-        IoUtil.deleteRecursive(new File(machineLogsDir));
+        machineLogsDir = targetDir().resolve("logs-dir").toString();
```

Comment:
What will happen if previous test is stopped before teardown method call and files removal haven't been accomplished?